manifests/init.pp

```diff
@@ -623,6 +634,7 @@
     $_agent_config = {
       'api_key' => $api_key,
       'dd_url' => $dd_url,
+      'site' => $datadog_site,
```

Comment:
I don't think the current manifest will yield a valid configuration for older A6 agents, unfortunately we need to support them: the problem we have here is that if this puppet manifestis used to install an older Agent6 (pre-`site` option) say `6.2.x` it will yield a configuration with an empty `dd_url` and `site` set. It causes no harm to have `site` set, but the empty `dd_url` would be a no-no.

Reply:
Unlike A5, A6 has a default value for `dd_url`: https://github.com/DataDog/datadog-agent/blob/60fc833599c5015c15c741649840dc6bda930275/pkg/config/config.go#L75. That's why a left this parameter empty

Reply:
Just tested, with an empty `dd_url`, agent 6.2.0 still reports to app.datadoghq.com

Reply:
This is accurate. Older agents *DID* specify a default for `dd_url` this was changed with the introduction of `site`, but this already accounts for that. 👍